meteor/client/ui/Settings/ConfigManifestSettings.tsx

```diff
@@ -85,19 +89,20 @@ function getTableColumnValues<DBInterface extends { _id: ProtectedString<any> }>
 	configPath: string,
 	object: DBInterface,
 	alternateObject?: any
-): string[] {
+): SelectOption[] {
 	const attribute = `${configPath}.${item.tableId}`
 	const table = objectPathGet(object, attribute) ?? objectPathGet(alternateObject, attribute)
-	const result: string[] = []
 	if (!Array.isArray(table)) {
-		return result
+		return []
 	}
-	table.forEach((row) => {
-		if (typeof row === 'object' && row[item.columnId] !== undefined) {
-			result.push(row[item.columnId])
-		}
-	})
-	return result
+	return table
+		.filter((row) => typeof row === 'object' && row[item.columnId] !== undefined)
+		.map((row) => {
+			return {
+				value: row['_id'],
+				label: row[item.columnId],
+			}
+		})
```

Comment:
You can remove return by using shorthand for object creation.

Reply:
Updated